pkg/util/util.go

```diff
@@ -1,6 +1,74 @@
 package util
 
-import "regexp"
+import (
+	"bytes"
+	"fmt"
+	"os/exec"
+	"regexp"
+	"strings"
+)
+
+const (
+	unknownPackage = "Unknown"
+)
+
+func queryPackageVersion(cmdArg ...string) string {
```

Comment:
Because you need to repush could you also add NOTE here from where you copied this functions.
If someone runs a git blame we loose all the history in this repo. Might be useful to point them to the original source if needed.